py23-image/redhat-8/Dockerfile

```diff
@@ -5,8 +5,9 @@ USER root
 RUN microdnf -y --nodocs update \
     && microdnf -y --nodocs install python2-pip python2-devel \
     && pip2 --no-cache-dir install requests pyyaml jmespath \
-    && ln -sf /usr/bin/python3.7 /usr/bin/python3 \
-    && ln -sf /usr/bin/pip3.7 /usr/bin/pip3 \
-    && ln -sf /usr/bin/python3.7 /usr/bin/python \
-    && ln -sf /usr/bin/pip3.7 /usr/bin/pip \
-    && pip3 install --upgrade ansible==3.4.0 requests==2.25.1 pyyaml==5.4.1 jmespath==0.10.0
+    && ln -sf /usr/bin/python3.8 /usr/bin/python3 \
+    && ln -sf /usr/bin/pip3.8 /usr/bin/pip3 \
+    && ln -sf /usr/bin/python3.8 /usr/bin/python \
+    && ln -sf /usr/bin/pip3.8 /usr/bin/pip \
+    && pip3 install --upgrade requests==2.25.1 pyyaml==5.4.1 jmespath==0.10.0 \
+    && sed -i '/^\[defaults\]/a\interpreter_python = /usr/bin/python3' /opt/ansible/ansible.cfg
```

Comment:
Without this, ansible tries to use `/usr/libexec/platform-python` (python3.6) for our custom modules like `splunk_api`.